python/ray/includes/gcs_client.pxi

```diff
@@ -653,7 +653,7 @@ cdef convert_get_all_actor_info(
         for b in serialized_reply:
             proto = gcs_pb2.ActorTableData()
             proto.ParseFromString(b)
-            actor_table_data[ActorID.from_binary(proto.actor_id)] = proto
+            actor_table_data[ActorID(proto.actor_id)] = proto
```

Comment:
let's just implement `from_binary` for ActorID.

Reply:
ok